cityofaustin/atd-vz-data
# Add new lookup tables and values to get VZDB up to date with CRIS v27

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/15263

Got to reuse @frankhereford code from https://github.com/cityofaustin/atd-vz-data/blob/refactor-vze-for-ldm/atd-toolbox/materials_from_ldm_conversion/inspect_lookup_table_data.py and tweaked it ever so slightly to create up and down migration files to easily copy/paste into the hasura console to create migrations. This is now a script that lives in our toolbox that can be run whenever there is a new CRIS release or we feel we aren't up to date.

## Testing
**URL to test:** 
Local

**Steps to test:**
1. Check out this branch locally
2. Get a fresh copy of prod data to run locally using the replicate-db command
3. Run the hasura up migrations
```bash
hasura migrate apply --up 2
hasura metadata apply
``` 
5. Run the hasura down migrations
```bash
hasura migrate apply --down 2
``` 
6. If you want you can even test the script yourself but it takes a really long time like at least 20 min i think? 
7. You can also just look through the script and make sure it makes sense, is readable etc.

---
#### Ship list
